### ISSUE TITLE
attr.label lacks allow_single_file

### ISSUE THREAD

**Issue by benley** [bazelbuild/skydoc#28]
Bazel added the `allow_single_file` attribute to attr.label at some point, and the older `single_file` one is now deprecated.  Skydoc breaks on rules using the new form:

```
Traceback (most recent call last):
  File "/private/var/tmp/_bazel_bstaffin/26abb02a80899f40a981f66c2d82ca06/execroot/ops/bazel-out/host/bin/external/io_bazel_skydoc/skydoc/skydoc.runfiles/com_fitbit_ops/../io_bazel_skydoc/skydoc/main.py", line 245, in <module>
    main(FLAGS(sys.argv))
  File "/private/var/tmp/_bazel_bstaffin/26abb02a80899f40a981f66c2d82ca06/execroot/ops/bazel-out/host/bin/external/io_bazel_skydoc/skydoc/skydoc.runfiles/com_fitbit_ops/../io_bazel_skydoc/skydoc/main.py", line 224, in main
    rule_doc_extractor.parse_bzl(bzl_file)
  File "/private/var/tmp/_bazel_bstaffin/26abb02a80899f40a981f66c2d82ca06/execroot/ops/bazel-out/host/bin/external/io_bazel_skydoc/skydoc/skydoc.runfiles/io_bazel_skydoc/skydoc/rule_extractor.py", line 179, in parse_bzl
    self._process_skylark(bzl_file)
  File "/private/var/tmp/_bazel_bstaffin/26abb02a80899f40a981f66c2d82ca06/execroot/ops/bazel-out/host/bin/external/io_bazel_skydoc/skydoc/skydoc.runfiles/io_bazel_skydoc/skydoc/rule_extractor.py", line 65, in _process_skylark
    exec(compiled) in SKYLARK_STUBS, skylark_locals
  File "tools/build_rules/build_data.bzl", line 41, in <module>
    allow_single_file = True,
TypeError: label() got an unexpected keyword argument 'allow_single_file'
```

**Comment by benley** [comment]
I see #26 addresses this...

**Comment by davidzchen** [comment]
Sorry for the wait. I have been a bit swamped recently. I have merged #26.
